rest-service/manager_rest/utils.py

```diff
@@ -454,3 +439,10 @@ def _get_timestamp(value):
         return date_parser.parse(value, ignoretz=True)
     else:
         return value
+
+
+def is_expired(expiry):
+    """Check whether an expiry time has passed."""
+    if isinstance(expiry, str):
```

Comment:
does this _really_ need a case for strings?

Reply:
I think it's passed in as a string when creating the token, but let me double check whether I fixed that.

Reply:
It's a string when it's used in the token auth. I could pull the logic out to there, but this way it'll work if passed a datetime or if passed a datetime-string formatted in the way we usually do, so I'm inclined to leave it in.